code/game/machinery/medical_kiosk.dm

```diff
@@ -246,7 +246,7 @@
 				var/bit_vol = bit.volume - belly.food_reagents[bit.type]
 				if(bit_vol > 0)
 					chemical_list += list(list("name" = bit.name, "volume" = round(bit_vol, 0.01)))
-	if(altPatient.reagents.addiction_list.len)
+	if(altPatient.reagents?.addiction_list?.len)
```

Comment:
will reagents ever be null?

Reply:
From what I can see, reagents is initialised as null, so _maybe_? I'm not 100% sure if its necessary, but it's there to be safe.

Reply:
This bit is gone anyways, lazylens galore!